src/lua/thread.rs

```diff
@@ -111,6 +111,12 @@ pub fn run_with_lua<F>(func: F) -> rlua::Result<()>
     }
 }
 
+fn emit_refresh(lua: &rlua::Lua) {
+    // FIXME: This assumes that the Lua code does not e.g. unset the global. Can we somehow emit
+    // the signal directly?
+    let _ = lua.exec::<()>("awesome.emit_signal('refresh')", None);
```

Comment:
Can you achieve this by calling `signal::global_emit_signal("refresh".to_owned(), ())`? That's what `awesome.emit_signal` is bound to, and since the signals emitted are part of the global signals list this should work.

Reply:
Thanks, that looks like exactly the right thing.

Reply:
Sigh...
```
error[E0603]: module `signal` is private
  --> src/lua/thread.rs:27:5
   |
27 | use super::super::awesome::signal;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Should I make it public?

Reply:
Yeah go ahead

Reply:
also might be easier at that point to do `use awesome::signal` and not have to go up two layers instead